kernel/rtlil.cc

```diff
@@ -1034,6 +1135,14 @@ namespace {
 					cell->name.c_str(), cell->type.c_str(), __FILE__, linenr, buf.str().c_str());
 		}
 
+		void is_param(const RTLIL::IdString& name)
```

Comment:
It's related, having a version of this function that doesn't have a return value prevents unpacking every single string attribute to bits. Without this, the PR wouldn't have any positive impact because once you ran check, all Const would unpack

Reply:
That's for a parameter not an attribute though

Reply:
Ok, removed